services/blockchain-app-registry/shared/utils/fsUtils.js

```diff
@@ -51,17 +51,25 @@ const mkdir = async (directoryPath, options = { recursive: true }) => {
 	);
 };
 
-const rmdir = async (directoryPath, options = { recursive: true }) => {
+const rmdir = async (directoryPath, options = { recursive: true }) => new Promise((resolve) => {
```

Comment:
Why did we need this change in the implementation?

Reply:
This method implementation was wrong. `fs.rmdir` is callback based, but we weren't utilising that. As a result the the function execution was being finished before files were deleted sometimes. Also, this function is deprecated and was getting warning log to replace with `fs.rm`